packages/react/src/products/hooks/useProductListing.ts

```diff
@@ -62,6 +63,17 @@ const useProductListing = (
     getProductsListProducts(state, productListingHash),
   );
 
+  const productsWithBrands = useSelector((state: StoreState) =>
```

Comment:
What if this was a redux selector as well, would it be useful?

Reply:
This was already a selector: `getBrand`, that was being used inside the hook.   
But I think I did understand what you meant. I've updated the `getProductsListProducts` selector to return the product's data instead of returning just the brand id.
What do you think?